github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```
@@ -373,6 +373,68 @@ periodics:
         privileged: true
     nodeSelector:
       type: bare-metal-external
+- annotations:
+    testgrid-dashboards: kubevirt-periodics
+  cluster: prow-workloads
+  cron: 0 22, 10 * * *
```

Not sure whether the cron parser will stumble over the whitespace before the comma?

```suggestion
  cron: 0 22,10 * * *
```

---

I wanted this job to have the same frequency as the sriov periodic so I just took the cron expression from the sriov lane so I think this should be fine as we have it in the configuration already. 
https://github.com/kubevirt/project-infra/blob/1c203bc199a78a2b56f2d47fd82fb54dc06454c2/github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml#L381